tools/extra/qpafilter/qpafilter.py

```diff
@@ -178,23 +210,25 @@ def filter(self, items):
 
 def get_filter(category):
     """Return the filter class, given a category name."""
-    filters = { TEMPERATURE_CATEGORY: temp_filter
-    }
+    filters = {TEMPERATURE_CATEGORY: temp_filter}
     return filters[category]
 
 
 class two_way_map:
+    """Given a dict of strings to integers, provide the ability
+       to perform a 2-way translation. That is, given a string
+       key, retrieve the integer value. Given an integer key,
+       return the string value."""
     def __init__(self, str_to_int):
         self.str_to_int = str_to_int
-        self.int_to_str = {v: k for k,v in str_to_int.items()}
+        self.int_to_str = {v: k for k, v in str_to_int.items()}
 
     def __getitem__(self, i):
         if isinstance(i, str):
             return self.str_to_int[i]
-        elif isinstance(i, int):
+        if isinstance(i, int):
```

Comment:
nice.. is it because the previous block would have returned  if it evaluated true already.  Hence, no need for elif here?

Reply:
Exactly.  The code flow is the same, but pylint is finicky about "unnecessary" elif's.